owl/EASE-OBJ.owl

```diff
@@ -2251,6 +2308,15 @@
     
 
 
+    <!-- http://www.ease-crc.org/ont/EASE-OBJ.owl#RelevantPart -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-OBJ.owl#RelevantPart">
```

Comment:
I would suggest this be named a bit differently so that it can't be confused with a part, especially that we seem to go towards a nice distinction between parts and features.